dev-util/gengetopt/gengetopt-2.23.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A tool for generating scanners"
+DESCRIPTION="Gengetopt is a tool to write command \
+line option parsing code for C programs."
+HOMEPAGE="https://www.gnu.org/software/gengetopt/"
+COPYRIGHT="1999-2019 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/gengetopt/gengetopt-$portVersion.tar.xz"
+CHECKSUM_SHA256="b941aec9011864978dd7fdeb052b1943535824169d2aa2b0e7eae9ab807584ac"
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2"
```

Comment:
x86_gcc2 is actually primary architecture (feature for x86_64 maybe but not ready), check other recipes on what they use